src/tasks/batch_integration/metrics/pcr/script.py

```diff
@@ -13,17 +13,14 @@
 ## VIASH END
 
 print('Read input', flush=True)
-adata = ad.read_h5ad(par['input_integrated'])
-
-
-print('preprocess data', flush=True)
-adata.X = adata.layers['normalized']
-adata_int = adata.copy()
+input_solution = ad.read_h5ad(par['input_solution'])
+input_integrated = ad.read_h5ad(par['input_integrated'])
+input_solution.X = input_solution.layers['normalized']
 
 print('compute score')
 score = pcr_comparison(
```

Comment:
Can a 'layer' argument be added to the `pcr_comparison` function? This would save us the trouble of having to copy the normalized data to `.X`.

Reply:
Alright, thanks. When the time comes, let's do this in a separate PR.